e2e/doc/code_layout.txt

```diff
@@ -0,0 +1,44 @@
+e2e
+├── doc: Various documents
+│   ├── code_layout.txt
+│   └── condition_table.md
+├── keywords: Keyword definitions for workloads
+│   │   ├── __init__.py
+│   │   ├── common_keywords.py
+│   │   ├── engine_keywords.py
+│   │   ├── node_keywords.py
+│   │   ├── pod_keywords.py
+│   │   ├── replica_keywords.py
+│   │   └── volume_keywords.py
+├── libs: Library folder developed in Python for keywords
+│   ├── __init__.py
+│   ├── keywords: Functions that implement keyword functionality
+│   │   ├── __init__.py
```

Comment:
Look like inconsistent with the current PR.

Reply:
Addressed. I've re-update the code structure with using `tree` command.